src/modules/geolocation.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable consistent-return */
```

Comment:
- [ Required ] I think it would be better if you try and solve this linter `error` instead of disabling it on this file. Linter checks help you as a developer to write code that follows best practices and it's therefore recommended that you don't disable them. Please rectify this.